lib/diagnostic.rb

```diff
@@ -108,10 +108,10 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = 'steve = Hash.new; steve[:favorite_number] = 12; steve[:first_name] = 'Steve';'
 
 # Set a default return value to your `person` hash that concatenates the value
 # of person[:first_name] and a message about them.
 # Write your response as a Ruby string.
 
-Diagnostic.hash_default = 'your answer here'
+Diagnostic.hash_default = 'steve = Hash.new(steve[:first_name] "is so icy cool")'
```

Comment:
This is with constructors, we wanted short hand syntax again:

`'"#{steve[:first_name]} is great"'`